### PR TITLE
Feat : Added Documentation

### PR DESCRIPTION
**Pull request by srejitk**
This feature request adds Documentation Sites for the following components

1. Alerts
2. Avatars
3. Badges
4. Buttons
5. Cards
6. Colors
7. Grids
8. Image
9. Inputs
10. Introduction
11. List
12. Rating
13. Text Utilities

### CODE REVIEW

File: docs/Avatars.html [hunk 1]
```diff
@@ -0,0 +1,248 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Iridescent</title>
```
Comment (Sameony) [inline docs/Avatars.html]: Add a relative title to the document

File: docs/Badges.html [hunk 2]
```diff
@@ -0,0 +1,317 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Alerts</title>
```
Comment (Sameony) [inline docs/Badges.html]: Replace the title with something relevant

File: docs/Avatars.html [hunk 3]
```diff
@@ -0,0 +1,248 @@
...
+                </div>
+            </nav>
+        </header>
+        <!-- Side bar HTML -->
+        <section id="sidebar" class="iris__sidebar">
+            <div class="section__title">Getting Started</div>
+            <a href="./Introduction.html" class="iris__categories title hoverOn ">Introduction</a>
+            <a href="./Colors.html" class="iris__categories title hoverOn">Colors</a>
+            <a href="./Typography.html" class="iris__categories title hoverOn">Typography</a>
+            <!-- <a href="./Contribution" class="iris__categories title hoverOn">Contribution</a> -->
```
Comment (tanishq20) [inline docs/Avatars.html:49]: Remove unused links before merging.

File: docs/Alerts.html [hunk 4]
```diff
@@ -0,0 +1,371 @@
...
+                            </div>
+                            <!-- Insert Component Label Here -->
+                            <span class="subtitle-1">Failure Alert</span>
+                        </div>
+
+                    </div>
+            </section>
+            <!-- Implementation Code Section -->
+            <section class="iris__desc__wrapper">
+                <h2 class="header-h2 content__title">Implementation</h2>
```
Comment (tanishq20) [inline docs/Alerts.html]: Have a cross-check on `header-h2` class

**Review by coldpigli — APPROVED** [review]
Exceptionally well written HTML code. Go ahead and merge

File: docs/Cards.html [hunk 5]
```diff
@@ -0,0 +1,477 @@
...
+                                <p class="body-2 pad-12side">Starting from 10k</p>
+                            </div>
+                            <span class="card__icon material-icons">close</span>
+                        </div></code></pre>
+                </div>
+
+
+            </section>
+            <!-- Page Navigation -->
+            <section class="iris__desc__wrapper page_nav">
```
Comment (Sameony) [inline docs/Cards.html]: use nav tag for navigation bars

File: docs/Text-Utils.html [hunk 6]
```diff
@@ -0,0 +1,289 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Typography</title>
```
Comment (Sameony) [inline docs/Text-Utils.html]: This and your typography have same title. please, see to it.

**Review by tanishq20 — COMMENTED** [review]
Good reuse of class name. Suggested minor change.

File: docs/Typography.html [hunk 7]
```diff
@@ -0,0 +1,285 @@
...
+    <link rel="stylesheet" href="/main.css">
+    <link href="https://fonts.googleapis.com/icon?family=Material+Icons" rel="stylesheet" />
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/highlight.js/11.4.0/styles/night-owl.min.css"
+        integrity="sha512-i5X6Fdn/ZqvGSqPrdMa3FgcpXM/Nr6YccSKFYT93zljl/HZDEpvBbE5Pxp91eiWGccZLrL/LDQJd7fjTRYsVaA=="
+        crossorigin="anonymous" referrerpolicy="no-referrer" />
+    <script>
+        hljs.highlightAll();
+    </script>
```
Comment (Sameony) [inline docs/Typography.html:18]: Same tag is used at the bottom of the file. see to it.

File: docs/inputs.html [hunk 8]
```diff
@@ -0,0 +1,513 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Iridescent</title>
```
Comment (Sameony) [inline docs/inputs.html]: Give relevant title

File: docs/inputs.html [hunk 9]
```diff
@@ -0,0 +1,513 @@
...
+    <link rel="stylesheet" href="/main.css">
+    <link href="https://fonts.googleapis.com/icon?family=Material+Icons" rel="stylesheet" />
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/highlight.js/11.4.0/styles/night-owl.min.css"
+        integrity="sha512-i5X6Fdn/ZqvGSqPrdMa3FgcpXM/Nr6YccSKFYT93zljl/HZDEpvBbE5Pxp91eiWGccZLrL/LDQJd7fjTRYsVaA=="
+        crossorigin="anonymous" referrerpolicy="no-referrer" />
+    <script>
+        hljs.highlightAll();
+    </script>
```
Comment (Sameony) [inline docs/inputs.html]: Remove duplicate code blocks

File: js/app.js [hunk 10]
```diff
@@ -0,0 +1,26 @@
...
+
+    sidebar.classList.remove("sidebar--open");
+    sidebarClose.style.display ="none";  
+    sidebarOpen.style.display ="block";  
+
+}
+
+
+  
+
```
Comment (Sameony) [inline js/app.js]: remove empty blocks

File: stylesheets/colors.css [hunk 11]
```diff
@@ -76,5 +76,20 @@
...
+		--primary-color-red: #DA100B;
+		--primary-text-red: #FCD0CF;
+
+
+}
+
+
+
+
+
```
Comment (Sameony) [inline stylesheets/colors.css]: remove empty blocks and refactor your code

File: stylesheets/content.css [hunk 12]
```diff
@@ -14,9 +28,9 @@
     margin: 2.5rem;
     padding: 2.5rem;
     border-radius: 64px;
-    outline: 2px solid;
-    height: 360px;
-  }
+    min-height: 360px;
```
Comment (Sameony) [inline stylesheets/content.css]: use rem for consistency

File: stylesheets/layout.css [hunk 13]
```diff
@@ -0,0 +1,23 @@
...
+}
+
+.hoverOn--round {
+    border-radius: 50px;
+
+}
+
+.hoverOn--round:hover {
+    border-radius: 50px;
+    background-color: #E3E3E3;
```
Comment (Sameony) [inline stylesheets/layout.css]: create color variables for better readability.

**Review by Sameony — APPROVED** [review]
Refactor and align your code properly before merging.

**Review by tanishq20 — APPROVED** [review]

File: docs/Avatars.html (hunk 3 above)
Comment (srejitk) [reply docs/Avatars.html:49]: Resolved in the latest commit. Thank You.

File: docs/Alerts.html (hunk 4 above)
Comment (srejitk) [reply docs/Alerts.html]: Resolved in the latest commit. Thank You.

File: docs/Avatars.html (hunk 1 above)
Comment (srejitk) [reply docs/Avatars.html]: Resolved in the latest commit. Thank You.

File: docs/Badges.html (hunk 2 above)
Comment (srejitk) [reply docs/Badges.html]: Resolved in the latest commit. Thank You.

File: docs/Cards.html (hunk 5 above)
Comment (srejitk) [reply docs/Cards.html]: Resolved in the latest commit. Thank You.

File: docs/Text-Utils.html (hunk 6 above)
Comment (srejitk) [reply docs/Text-Utils.html]: Resolved in the latest commit. Thank You.

File: docs/inputs.html (hunk 9 above)
Comment (srejitk) [reply docs/inputs.html]: Resolved in the latest commit. Thank You.

File: js/app.js (hunk 10 above)
Comment (srejitk) [reply js/app.js]: Resolved in the latest commit. Thank You.

File: stylesheets/colors.css (hunk 11 above)
Comment (srejitk) [reply stylesheets/colors.css]: Resolved in the latest commit. Thank You.

File: stylesheets/content.css (hunk 12 above)
Comment (srejitk) [reply stylesheets/content.css]: Resolved in the latest commit. Thank You.

**Pull request merged**